.tekton/roxctl-pull-request.yaml

```
@@ -8,7 +8,7 @@ metadata:
     build.appstudio.redhat.com/target_branch: '{{target_branch}}'
     pipelinesascode.tekton.dev/max-keep-runs: "3"
     pipelinesascode.tekton.dev/on-event: '[pull_request]'
-    pipelinesascode.tekton.dev/on-target-branch: '[*]'
+    pipelinesascode.tekton.dev/on-target-branch: '[master]'
```

For `.tekton/roxctl-pull-request.yaml` I suggest to keep `pipelinesascode.tekton.dev/on-target-branch: '[*]'`, otherwise PRs for non-master branches don't have builds, see https://github.com/stackrox/stackrox/pull/8324

```suggestion
    pipelinesascode.tekton.dev/on-target-branch: '[*]'
```